adapter.go

```diff
@@ -94,7 +94,7 @@ func (a *Adapter) createDatabase() error {
 				return nil
 			}
 		}
-	} else {
+	} else if a.driverName != "sqlite3" {
```

Comment:
Why don't create DB for sqlite3?

Reply:
Because in sqlite there is no such thing as databases, only tables. In sqlite the file is the database. 

In something like mysql, you have a mysql server which has databases, which has tables. 

In sqlite there is no such thing as a server that has databases, the file on disk or memory is the database. See: https://www.sqlite.org/quickstart.html and https://www.sqlite.org/cli.html

Reply:
I know sqlite database is file on disk. I mean the creating DB in Xorm here won't create the DB file on local disk, right?

Reply:
sqlite will create the file on disk if it does not exist according to the docs. `file:test.db?cache=shared&mode=memory` this is the DNS, so sqlite will create test.db if it does not exist,

Reply:
ok